orchestration/dagster_orchestration/hca_orchestration/contrib/argo_workflows.py

```diff
@@ -0,0 +1,67 @@
+from argo.workflows.client import ApiClient as ArgoApiClient,\
+                                  ArchivedWorkflowServiceApi,\
```

Comment:
Did this not get added? I can't find the class definition for ArchivedWorkflowServiceApi when I search in this PR

Reply:
Ah nm I see that's coming from a lib dependency.